packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -13,6 +13,7 @@ import { useSelector } from 'react-redux';
 import { getUserPurchases } from 'calypso/state/purchases/selectors';
 import { getSectionName, getSelectedSiteId } from 'calypso/state/ui/selectors';
 import NewReleases from '../icons/new-releases';
+export const SITE_STORE = 'automattic/site';
```

Comment:
I can't seem to find where this exported constant is used, I've checked for this file and any other file in Calypso with no luck.

Reply:
Removed